### ISSUE TITLE
Can't Drag Slimes Into Scanner

### ISSUE THREAD

**Issue by Kurfursten** [d3athrow/vgstation13#30925]
This is supposed to be a feature, there's code for it in the cloning computer. Couldn't find any way to get a slime pile into the scanner machine, nor is there code for it seemingly.

Also if you clickdrag to unload the dna scanner you get the message "The SLEEPER is unoccupied"

**Comment by DeityLink** [comment]
> This is supposed to be a feature, there's code for it in the cloning computer. Couldn't find any way to get a slime pile into the scanner machine, nor is there code for it seemingly.

So you're telling me that despite people telling others for years to do it and me even tweaking the description of slime piles, you can't actually drag slime piles into DNA scanners? bruh.

**Comment by Dacendeth** [comment]
You used to be able to yellow intent them into the cloner, like you can with any corpse, but now attempting to grab the puddle just rips their brain out.

**Comment by Kurfursten** [comment]
Yeah I explicitly tested yellow intent grab and it just pulls out the slime core (or if core is removed gives a message that there is no core to remove)

**Comment by SECBATON-GRIFFON** [comment]
clickdragging in and out worked for me
